test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,88 @@
+SCOPE="https://www.googleapis.com/auth/cloud-platform"
+REGION=us-central1
+ZONE=${REGION}-b
+PROJECT=spheric-keel-331521
+
+#Configure Kubernetes
+echo -e "==================== Prepare environments with set of environment variables "
+echo -e "==================== Set Kubernetes Cluster "
+export CLUSTER=$CLUSTER
+echo -e "==================== Set Kubernetes TARGET "
+export TARGET=$TARGET
+echo -e "==================== Set SCOPE "
+export SCOPE=$SCOPE
+
+echo -e "==================== Prepare GCP by updating default zone and project ID "
+$GCLOUD config set compute/zone $ZONE
+$GCLOUD config set project $PROJECT
+
+echo -e "==================== Refresh Kubeconfig at path ~/.kube/config "
+$GCLOUD container clusters get-credentials $CLUSTER --zone $ZONE --project $PROJECT
+
+##Build Docker Images
+echo -e "==================== Build the Docker image and store it in your project's container registry. Tag with the latest commit hash "
+$GCLOUD builds submit --tag gcr.io/$PROJECT/locust-tasks:$LOCUST_IMAGE_TAG
+echo -e "==================== Verify that the Docker image is in your project's container repository"
+$GCLOUD container images list | grep locust-tasks
+
+##Deploying the Locust master and worker nodes
+echo -e "==================== Update Kubernetes Manifests "
+echo -e "==================== Replace the target host and project ID with the deployed endpoint and project ID in the locust-master-controller.yml and locust-worker-controller.yml files"
+$SED -i -e "s/\[TARGET_HOST\]/$TARGET/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[TARGET_HOST\]/$TARGET/g" $MERINO_DIRECTORY/$WORKER_FILE
+$SED -i -e "s/\[PROJECT_ID\]/$PROJECT/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[PROJECT_ID\]/$PROJECT/g" $MERINO_DIRECTORY/$WORKER_FILE
+$SED -i -e "s/\[LOCUST_IMAGE_TAG\]/$LOCUST_IMAGE_TAG/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[LOCUST_IMAGE_TAG\]/$LOCUST_IMAGE_TAG/g" $MERINO_DIRECTORY/$WORKER_FILE
+$SED -i -e "s/\[KINTO_BUCKET\]/$KINTO_BUCKET/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[KINTO_BUCKET\]/$KINTO_BUCKET/g" $MERINO_DIRECTORY/$WORKER_FILE
+$SED -i -e "s/\[KINTO_COLLECTION\]/$KINTO_COLLECTION/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[KINTO_COLLECTION\]/$KINTO_COLLECTION/g" $MERINO_DIRECTORY/$WORKER_FILE
+$SED -i -e "s/\[KINTO_SERVER_URL\]/$KINTO_SERVER_URL/g" $MERINO_DIRECTORY/$MASTER_FILE
+$SED -i -e "s/\[KINTO_SERVER_URL\]/$KINTO_SERVER_URL/g" $MERINO_DIRECTORY/$WORKER_FILE
+
+$KUBECTL delete pods --all
+$KUBECTL delete svc locust-master
+
+##Deploy the Locust master and worker nodes using Kubernetes Manifests
+echo -e "==================== Deploy the Locust master and worker nodes"
+$KUBECTL apply -f $MERINO_DIRECTORY/$MASTER_FILE
+$KUBECTL apply -f $MERINO_DIRECTORY/$SERVICE_FILE
+$KUBECTL apply -f $MERINO_DIRECTORY/$WORKER_FILE
+
+echo -e "==================== Verify the Locust deployments & Services"
+$KUBECTL get pods -o wide
+$KUBECTL get services
+sleep 30s
```

Comment:
Reviewer note: The <kbd>sleep</kbd> is here because it takes the pods a little while to warm up and deploy and we get an empty `$EXTERNAL_IP` if we try calling the <kbd>kubectl get svc locust-master</kbd> command below too soon before the load balancer ingress IP is available. :shrug: 🕐